### PR TITLE
added day 5 resolved

### PR DESCRIPTION
**Pull request by syedowaisalishah**

### CODE REVIEW

**Comment by Talha-Ahmed-1** [comment]
Resolve the conflicts..

**Comment by Talha-Ahmed-1** [comment]
Is it proper way to place comma (,) in filenames ? Is the filename meaningful ?

**Comment by Talha-Ahmed-1** [comment]
Why do you make changes in day3 files?

File: src/main/scala/day4/conditions.scala [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package day4
+import chisel3._
+import chisel3.util._
+class condition extends Module{
```
Comment (Talha-Ahmed-1) [inline src/main/scala/day4/conditions.scala:4]: place a line break between class and import, and capitalize the class and file name.

File: src/test/scala/day4/conditions.scala [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package day4
+import chisel3._
+import org.scalatest._
+import chiseltest._
+class whentest extends FreeSpec with ChiselScalatestTester{
```
Comment (Talha-Ahmed-1) [inline src/test/scala/day4/conditions.scala:5]: Please use same class name for test as module name simply place test after that, and capitalize the names.

**Comment by Talha-Ahmed-1** [comment]
@samadpls help him resolving the git related issues.

**Comment by samadpls** [comment]
> @samadpls help him resolving the git related issues.

Okk ✌️

**Comment by Talha-Ahmed-1** [comment]
Why do you have update files in day 3?

**Comment by samadpls** [comment]
> Why do you have updated files on day 3?

Sir, he accidentally gave space in the day3 code…

**Pull request merged**